terraform/modules/services/s3-virus-scan/s3_virus_scan.json.tpl

```diff
@@ -13,6 +13,20 @@
           "awslogs-stream-prefix": "ecs"
         }
     },
+    "environment": [
+      {
+        "name": "AWS_REGION",
+        "value": "${aws_region}"
+      },
+      {
+        "name": "AWS_ACCESS_KEY_ID",
+        "value": "${aws_access_key_id}"
+      },
+      {
+        "name": "AWS_SECRET_ACCESS_KEY",
```

Comment:
Can we make these ECS container secrets instead of plain text environment variables?